### PR TITLE
add search engine `pkg.go.dev`

### PR DESCRIPTION
**Pull request by lunarxlark**
Please add search engine for Gopher, if you like.

### CODE REVIEW

File: doc/openbrowser.txt [hunk 1]
```diff
@@ -249,6 +249,7 @@ g:openbrowser_search_engines		*g:openbrowser_search_engines*
 		\       'devdocs': 'http://devdocs.io/#q={query}',
 		\       'duckduckgo': 'http://duckduckgo.com/?q={query}',
 		\       'github': 'http://github.com/search?q={query}',
+		\       'go': 'https://pkg.go.dev/search?q={query}'
```
Comment (tyru) [inline doc/openbrowser.txt]: missing comma at the end

Comment (lunarxlark) [reply doc/openbrowser.txt]: thx

**Comment by tyru** [comment]
Okay, thanks.

**Comment by lunarxlark** [comment]
@tyru Thank you for merge!

**Pull request merged**